components/RepoListView.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { FlatList, StyleSheet } from 'react-native'
+import { ListItemView } from './ListItemView'
+
+interface Props {
+    data: any[];
```

Comment:
The type this data should be a object with these fields {
    Repository name
    Repository description
    Numbers of stars for the repo.
    Username and avatar of the owner.
}, and should be created in a file on the project root named: types.d.ts

Reply:
ok, but this is only a mock data, the next step will be make a query that retreive the real data.

Reply:
ok, but for the type of the data, remember to create an object with the feature that I wrote you.